homework/10_store_conditions.md

```diff
@@ -0,0 +1,28 @@
+# Homework 10 -- Store
+
+Keep working on your little online store prototype and practice conditions.
+
+> Important: do not forget to do your homework on a separate branch with meaningful name
+
+## Spec
+
+We going to add some extra functionality to your online store.
+
+1. Add an "amount" property to each product.
```

Comment:
I like the homework but is it enough for a week? I think we should challenge them more.

Reply:
I think this HW will take few hours to complete for them anyway. And many students are still struggling with previous homeworks..